src/commands/admin/issue.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable camelcase */
+import { adminPerms } from '@lib/permissions';
+import { RequestError } from '@octokit/types';
+import { Message } from 'discord.js';
+
+export const usage = '<title>|[options]';
+export const extendedHelp = `You must pass in an issue title. Flags can be any of:
+--labels=[comma, separated, list]\n--milestone=[milestone number]\n--project=[project (defaults to SageV2)]]`;
+
+export async function permissions(msg: Message): Promise<boolean> {
+	return adminPerms(msg);
+}
+
+export async function run(msg: Message, [title, project, labels, milestone]: [string, string, string[], string]): Promise<Message | void> {
+	const newIssue = await msg.client.octokit.issues.create({
+		owner: 'ud-cis-discord',
+		repo: project,
+		title: title,
+		milestone: milestone !== '' ? milestone : undefined,
```

Comment:
Can probably be simplified to...
```suggestion
		milestone: milestone || undefined,
```